cypress/support/index.js

```diff
@@ -18,7 +18,7 @@ import './data-import';
 import './commands';
 
 
-setInteractorTimeout(20_000);
+setInteractorTimeout(30_000);
```

Comment:
You need ask another part of team to change this file

Reply:
thanks for advice

Reply:
changed to 80K)